spring-integration-core/src/main/java/org/springframework/integration/handler/DelayHandler.java

```diff
@@ -689,7 +692,8 @@ public static final class DelayedMessageWrapper implements Serializable {
 		@SuppressWarnings("serial")
 		private final Message<?> original;
 
-		DelayedMessageWrapper(Message<?> original, long requestDate) {
+		@JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+		DelayedMessageWrapper(@JsonProperty("original") Message<?> original, @JsonProperty("requestDate") long requestDate) {
```

Comment:
Our preference the line length not more than 120 symbols.

Reply:
It has been handled. But the code style confuses me. I use spring-javaformat to format my code in daily development. This project provides me with [idea code configuration](https://github.com/spring-projects/spring-integration/blob/main/src/idea/spring-framework.xml), but I found that after importing IDEA, the formatted code is inconsistent with the existing one, which causes me to often adjust the code manually

Reply:
Right. It might not be consistent with config everywhere: too much code to reformat at once. My IDEA just crashes. However it is fine with me to reformat files as things move on in them .  However I don’t think it is able to break on 120 length.
I’m not very fond with `spring-javaformat`: sometimes it squashs method chain into one line, or makes lines more than 120. 
However I’m open to move there as well if that is what community expects from us.